Translate documents to french (#10)
team-gsri/.github: CODE_OF_CONDUCT_FR.md

***Note :** ce document est une traduction du [GSRI Code of Conduct](./CODE_OF_CONDUCT.md). En cas d'incohérence entre les deux documents, c'est la version anglophone qui s'applique. Cette version est fournie pour faciliter la compréhension mais n'a pas la valeur probante du document anglophone.*

## Qui sommes-nous ?

Le `Groupe Special de Reconnaissance et d'Intervention` est une équipe ArmA 3 française, orientée milsim. Nous sommes une organisation sans but lucratif composée de membres bénévoles, qui contribuent au développement d'outils et de mods pour le jeu Arma 3.

Nous jouons des missions, privées comme publiques, orientées forces spéciales, à un haut niveau de discipline et de réalisme. Davantage d'information est disponible sur notre [site internet officiel](https://www.gsri.team) et sur notre [serveur Discord](https://discord.gg/bhMn4jd). L'un comme l'autre sont consultables uniquement en français.

## Notre engagement

> Dans le but de favoriser un environnement ouvert et accueillant, nous, contributeurs, nous engageons à faire de notre participation dans notre projet et notre communauté une expérience libre de tout harcelement pour tout invididu.

Reviewer: ```suggestion
Dans le but de favoriser un environnement ouvert et accueillant, nous, contributeurs, nous engageons à faire en sorte que la participation de chacun dans nos projets et notre communauté ne soit pas source de harcèlement.
```

## Note importante sur le contenu violent

> Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté entièrement publique.

Reviewer: ```suggestion
Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté tout public.
```

Author: If I'm right, there is no change in this suggestion ?

> Si vous vous souciez du fait que vous ou vos enfants puissiez être exposé à un tel contenu, nous vous encourageons à vous éloignez de notre contenu, ou à faire le nécessaire pour éviter à vos enfants de consulter notre contenu.

Reviewer: ```suggestion
Si vous vous êtes gênés par la possibilité que vous ou vos enfants puissiez être exposé à un tel contenu, nous vous encourageons à vous en éloigner, ou à faire le nécessaire pour éviter que vos enfants y accèdent.
```

## Règles de bonne conduite

### Comportement respectueux

> * **Soyez poli.** De simples expressions telles que *Bonjour*, *S'il-vous-plait*, *Merci*, ou encore *Au revoir* contribuent grandement à faire de cet espace un lieu accueillant et inclusif.

Reviewer: ```suggestion
* **Soyez poli.** De simples expressions telles que *Hello*, *Please*, *Thank you*, ou encore *Good bye* contribuent grandement à faire de cet espace un lieu accueillant et inclusif.
```

> * **Surveillez vos écrits.** Les correcteurs orthographiques sont de nos jours très communs. Commettre de trop nombreuses fautes vous fera apparaître comme nonchalant et irrespectueux. Utilisez la ponctuation pour faciliter la lecture de vos messages. Evitez l'emploi de textes en majuscules, qui seront interprétés comme si vous hurliez, et considérés grossiers. Espacez vos paragraphes mais n'abusez pas des sauts de ligne.

Reviewer: ```suggestion
* **Soyez attentif à votre rédaction.** Les correcteurs orthographiques sont de nos jours très communs. Commettre de trop nombreuses fautes vous fera apparaître comme nonchalant et irrespectueux. Utilisez la ponctuation pour faciliter la lecture de vos messages. Évitez l'emploi de textes en majuscules, qui seront interprétés comme si vous hurliez, et considérés comme grossier. Aérez vos contenus mais n'abusez pas des espacements.
```

* **Ne faites pas de hors-sujet.** Chaque espace de notre communauté est prévu pour un unique sujet. Si vous souhaitez référencer plusieurs sujets, créez un nouveau fil de discussion, et ayez recours au [Github Autolink](https://help.github.com/en/articles/autolinked-references-and-urls) pour les référencer.

> * **Terminez vos conversations.** En débuter une et ne plus répondre est considéré comme grossier, aussi pensez à le signaler lorsque vous n'avez plus l'intention de suivre l'évolution d'une discussion. Cela aidera l'équipe de modération à catégoriser les conversations et les lecteurs à savoir si un auteur attend une réponse ou non.

Reviewer: ```suggestion
* **Terminez vos conversations.** En débuter une et ne plus répondre est considéré comme grossier, aussi pensez à signaler lorsque vous n'avez plus l'intention de suivre l'évolution d'une discussion. Cela aidera l'équipe de modération à catégoriser les conversations, et les lecteurs à savoir si les conversations peuvent être poursuivies ou non.
```

> * **Abordez les conflits avec respect.** Le conflit est un phénomène social naturel, qui peut permettre le progrès s'il est traité correctement. Evitez de résoudre des conflits personnels en public. Astreignez-vous à n'argumenter que sur le fond du sujet et n'employez pas d'argument _ad hominem_. Les messages personnels rendus publics dans le but de discréditer une autre personne seront supprimés sans avertissement. Si une personne vous atteint en privé, vous devez [la bloquer](https://help.github.com/en/articles/blocking-a-user-from-your-personal-account). Notre équipe d'administration est à votre disposition si vous souhaitez une assistance afin de vous protéger d'autrui.

Reviewer: ```suggestion
* **Abordez les conflits avec respect.** Le conflit est un phénomène social naturel, qui peut permettre de s’améliorer s'il est traité correctement. Évitez de résoudre des conflits personnels en public. Ne détournez pas la conversation de l'objet du conflit et n'employez pas d'_ad hominem argumentum_. Les messages personnels rendus publics dans le but de discréditer une autre personne seront supprimés sans avertissement. Si une personne vous persécute en privé, vous devez [la bloquer](https://help.github.com/en/articles/blocking-a-user-from-your-personal-account). Notre équipe d'administration est à votre disposition si vous souhaitez de l'aide pour vous protéger des agressions.
```

### Bonnes pratiques pour la contribution

* **Communiquez en anglais.** Nous sommes une communauté francophone, mais notre objectif est de contribuer à la communauté ArmA dans son ensemble. L'usage de l'anglais permet aux joueurs d'autres pays et parlant différentes langues de réutiliser nos outils et contribuer à leur amélioration.

* **Ne surchargez pas les conversations.** Ne répondez que si vous avez du contenu à proposer. Si vous souhaitez réagir aux messages d'autres contributeurs, vous devez avoir recours aux émojis. Nous utilisons la convention suivante :
  * :+1: `:+1:` signifie **oui / d'accord**
  * :-1: `:-1:` signifie **non / en désaccord**
  * :hand: `:hand:` signifie **je suis volontaire**
  

> * **Respectez le style de programmation.** Les directives officielles comme informelles permettent de préserver la lisibilité et facilitent la maintenance.

Reviewer: ```suggestion
* **Respectez le style de programmation.** Les pratiques officielles comme informelles permettent de préserver la lisibilité et facilitent la maintenance.
```

* **Commentez votre code.** Soyez toutefois raisonnable sur la quantité de commentaires, souvenez-vous que le code concerne les *comment ?* tandis que les commentaires répondent aux *pourquoi ?*. Si votre code ne contient aucun commentaire, ou à l'inverse en contient beaucoup, vous faites probablement quelque chose de travers.

> * **Agissez en cohérence avec la communauté.** Cela permet d'éviter les gaspillages d'énergie sur un sujet déjà en cours de traitement, ainsi que de prioriser les chantiers les plus critiques. Le meilleur espace pour s'organiser est le [serveur Discord officiel du GSRI](https://discord.gg/bhMn4jd).

Reviewer: ```suggestion
* **Coordonnez-vous avec la communauté.** Cela permet d'éviter les gaspillages d'énergie sur un sujet déjà en cours de traitement, ainsi que de prioriser les chantiers les plus critiques. Le meilleur espace pour s'organiser est le [serveur Discord officiel du GSRI](https://discord.gg/bhMn4jd).
```

> * **Assurez-vous toujours de la compatibilité des licences.** Nos projets sont protégés par différentes licences, en fonction de la nature et de l'objectif du produit. Assurez-vous d'avoir pris connaissance de la licence utilisée avant de contribuer. Si vous souhaitez inclure du contenu tier dans vos contributions, vérifiez que la licence du projet est compatible avec votre contenu. Dans le doute, n'hésitez pas à solliciter un avis.

Reviewer: ```suggestion
* **Assurez-vous toujours de la compatibilité des licences.** Nos projets sont protégés par différentes licences, en fonction de la nature et de l'objectif du contenu. Assurez-vous d'avoir pris connaissance et d'avoir accepté les termes de la licence utilisée avant de contribuer. Si vous souhaitez inclure du contenu tiers dans vos contributions, vérifiez que la licence du projet est compatible avec celle qui le protège. Dans le doute, n'hésitez pas à solliciter un avis.
```

## Portée

Ce code de bonne conduite s'applique à tous les projets hébergés sur Github par l'organisation `gsri-team`.

## Application

> La violation du présent code de bonne conduite donnera lieu à une réaction jugée appropriée par l'équipe de modération, et notament :

Reviewer: ```suggestion
La violation du présent code de bonne conduite donnera lieu à une réaction jugée appropriée par l'équipe de modération, et notamment :
```

* Rappel à l'ordre
* Suppression de contenu
* Blocage temporaire
* Bannissement permanent
 
Nous ne rendons pas compte des actions de modération. Les rappels à l'ordre seront fait publiquement, nous n'aurons aucune conversation privée sur le sujet. Tout contenu questionnant la modération sera supprimé sans avertissement. Les remarques qualifiant la modération de trop zélée ou à l'inverse laxiste, conduiront à un bannissement immédiat et permanent.

## Signaler une violation

> Le meilleur espace pour prendre contact avec nous est [notre serveur Discord](https://discord.gg/bhMn4jd). Vous pouvez cliquer sur ce lien ou visister [notre site web www.gsri.team](https://www.gsri.team) où un lien permanent vers Discord est affiché au dessus de toutes nos pages. Les violations peuvent être signalées dans n'importe quel canal écrit, bien que nous vous encouragions à privilégier le canal `#discussions`. Merci de mentionner `@Cadre` (rassemblant les responsables de l'équipe) de manière à nous transmettre l'information aussi vite que possible. Les canaux vocaux sont également disponibles.

Reviewer: ```suggestion
Le meilleur espace pour prendre contact avec nous est [notre serveur Discord](https://discord.gg/bhMn4jd). Vous pouvez cliquer sur ce lien ou visister [notre site web www.gsri.team](https://www.gsri.team) où un lien permanent vers Discord est affiché au dessus de toutes nos pages. Les violations peuvent être signalées dans n'importe quel canal écrit, bien que nous vous encouragions à privilégier le canal `#discussions`. Merci de mentionner `@Cadre` de manière à nous transmettre l'information aussi vite que possible. Les canaux vocaux sont également disponibles.
```
